MIP4/mip4.md

```diff
@@ -59,22 +66,31 @@ MIP Amendments are invalid if, based on the assessment of the community, the cha
 **Removals**
 MIP4 also enables the removal of MIPs that are no longer useful. If there are other MIPs that depend on a MIP that is being removed, they must also be removed in the same governance cycle, or the proposal will be invalid.
 
----
-### MIP4c2: MIP Amendment Process
+### MIP4c2: MIP Amendment Processes
 
-MIP4c2 is a Process MIP component that allows the amendment of an Accepted MIP using a subproposal. MIP4c2 subproposals have the following parameters:
-- **Default Feedback Period**: 3 months
+MIP4c2 is a Process MIP component that allows the amendment of one or multiple Accepted MIPs, using a subproposal. MIP4c2 subproposals are done in two ways:
```

Comment:
Maybe these should be separate components processes to make the difference more readily visible for governance? I'd be tempted to attach a status to accepted-for-less-than-3-months MIPs. Provisionally Accepted versus Accepted?

Reply:
The template would be the same even if we made two components to handle amendments. 

I feel like its extra complexity and work for not much purpose or utility.

If we did

c2 MIP Amendment Process - Older than 3Mo
c3 MIP Amendment Process - Younger than 3Mo
c4 MIP Removal Process

Then it would seem a bit silly to me. Also the original c3 would be moved to c4, and subproposals would all be amended and renamed.

Otherwise it would be 

c2 MIP Amendment Process - Older than 3Mo
c3 MIP Removal Process
c4 MIP Amendment Process - Younger than 3Mo

But anyway, it would be slightly more accurate as an archive since we'll see which MIPs got amended early on and why just by creating the second component. hmm

Given those thoughts, do you still think it's a good idea to make it into a second component? We can work on the names of them.

MIP Amendment Process <3Months
MIP Amendment Process >3Months

Reply:
Considering it's done under the weekly cycle, the template should maybe be different in some way? Like the idea is that we can change these using the signaling process, right? Do you need to do a subproposal as well?

Reply:
I thought the idea was to still use a subproposal to detail what is being changed and the reasoning for why. It would go through a ["Non-Standard Weekly Poll"](https://github.com/makerdao/mips/blob/d37018aea81095d17a2a3cef48b26b0375e0d826/MIP16/mip16.md#mip16c1-weekly-governance-cycle-definitions) then be bundled in the next Executive with the other weekly stuff.

A Signal Thread would work as an expedited RFC process and could be put in as a requirement before getting a Gov Poll put up.

If you agree, I will add it

Reply:
Yeah, sounds good. Subproposal + Signal